notebooks/06 tsfresh on tsfresh.ipynb

```diff
@@ -0,0 +1,2169 @@
+{
```

Comment:
This needs some tweaks as some of the function signatures will most likely have been modified